packages/moonstone/BodyText/BodyText.js

```diff
@@ -48,18 +64,49 @@ const BodyTextBase = kind({
 		 * @type {Object}
 		 * @public
 		 */
-		css: PropTypes.object
+		css: PropTypes.object,
+
+		/**
+		 * Toggles multi-line (`false`) vs single-line (`true`) behavior. `nowrap` mode
+		 * automatically enables {@link moonstone/Marquee} so long text isn't permanently occluded.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		nowrap: PropTypes.bool
```

Comment:
Why did you choose to use all lower case vs. camel case for this prop?

Reply:
Habit from CSS. :facepalm: I did think about it, and slipped my mind that we allow camel case in prop names. I'll change this.